engine/src/main/java/org/hibernate/search/engine/common/impl/MappedIndexSearchTargetImpl.java

```diff
@@ -91,7 +112,7 @@ public String toString() {
 			Function<SearchQuery<T>, Q> searchQueryWrapperFactory,
 			SearchProjection<?>... projections) {
 		HitAggregator<ProjectionHitCollector, List<T>> hitAggregator =
-				new ProjectionHitAggregator<>( documentReferenceTransformer, objectLoader, hitTransformer,
+				new MultipleProjectionHitAggregator<>( documentReferenceTransformer, objectLoader, hitTransformer,
```

Comment:
In this case, shouldn't it be `MultipleProjectionsHitAggregator`? It's weird to have Multiple and a singular word.

Reply:
I thought of it like "a three-child family" (which is grammatically correct). But whatever, I'll change it.